core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -444,6 +466,7 @@ public void cleanupSession(long sessionId) {
 
   @Override
   public boolean hasBlockMeta(long blockId) {
+    LOG.debug("hasBlockMeta: blockId={}", blockId);
     try (LockResource r = new LockResource(mMetadataReadLock)) {
       return mMetaManager.hasBlockMeta(blockId);
     }
```

Comment:
should we add the debug logging to all the interface methods?

Reply:
I believe I added to all but two --- these two are very noisy.

Reply:
can we comment about this in the code so that we remember in the future?

Reply:
done